architecture/modules/choosing-registry.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+//
+// * architecture/understanding-openshift-development.adoc
+
+[id="choosing-registry_{context}"]
+= Choosing a registry
+
+Container Registries are where you store container images so you can share them with others and make them available to the platform where they ultimately run. There are large, public container registries that offer free accounts, as well as premium versions that offer more storage and special features. You can also install your own registry that can be exclusive to your organization or selectively shared with others.
+
+To get Red Hat images and certified partner images, you can draw from the Red Hat Registry. The Red Hat Registry is represented by two locations: registry.access.redhat.com (unauthenticated and deprecated) and registry.redhat.io (requires authentication). You can learn about the Red Hat and partner images in the Red Hat Registry from the https://www.google.com/url?q=https://access.redhat.com/containers/&sa=D&ust=1557950770715000[Red Hat Container Catalog]. Besides listing Red Hat container images, it also shows extensive information about the contents and quality of those images, including health scores based on applied security updates.
```

Comment:
@vikram-redhat, do we want to encourage people to use registry.access.redhat.com? I thought we'd fully switched to registry.redhat.io.